src/Backoffice/Sales/Clients/Domain/Client.php

```
@@ -0,0 +1,116 @@
+
+        $client = new self(
+            $id,
+            $code,
+            $name,
+            $description,
+            $status,
+            $companyId
+        );
+
+        $client->record(new ClientCreatedDomainEvent(
+            $client->id(),
```

PHPStan (2.0.3)

Parameter #1 $aggregateId of class MedineTech\Backoffice\Sales\Clients\Domain\ClientCreatedDomainEvent constructor expects string, string|null given.


(argument.type)